### ISSUE TITLE
Enhancement: API

### ISSUE THREAD

**Issue by GGPay** [my8100/scrapydweb#65]
Hi guys,

Are you going to add API for spiders? I think it might be really useful

**Comment by my8100** [comment]
Yes, RESTful APIs like [Scrapyd JSON API](https://scrapyd.readthedocs.io/en/latest/api.html) would be supported in a future release,
as well as the feature of interacting with ScrapydWeb in Python code.

**Comment by EchoShoot** [comment]
it's truly indeed, I really need api to deploy 'Timer Tasks', it’s too slow to use mouse.
